.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -50,3 +50,19 @@ body:
       placeholder: "# Your code here"
     validations:
       required: true
+  - type: textarea
+    id: error
+    attributes:
+      label: Errors, Traceback, and Logs
+      description: |
+        If applicable, share with us the Error and full Traceback provided by your interpreter.
+      render: python
```

Comment:
You can actually use `pytb` for tracebacks.
```suggestion
      render: pytb
```

Also, right now this looks like it's joining the lines not like code.

Reply:
Didn't know that; much appreciated. What do you mean re line joining?

Reply:
This:
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/221526/140993549-08018ca6-3bc2-49f5-aba6-568e6ad19830.png">

Not how lines that are separate in the traceback are joined. I'm hoping there's some magic somewhere that can fix that?

Reply:
Harumph, looks like `|` is supposed to [give us that](https://yaml-multiline.info). Might just be a limitation of GitHub's rendering of the form.

Reply:
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/24628426/140994751-b463962d-a6f1-4ed3-984b-afe6b27dba54.png">

Looks like that might just be a safari issue 😉

Reply:
🤦‍♂️